release-notes/opensearch-release-notes-3.0.0-alpha1.md

```diff
@@ -0,0 +1,790 @@
+# OpenSearch and OpenSearch Dashboards 3.0.0-alpha1 Release Notes
+
+## Release Highlights
+* Lucene 10 is now used in OpenSearch 3.0.0-alpha1
+* This is an early-stage preview of the 3.0.0 version, so expect potential bugs and unfinished features. This release is for testing purposes only, and we highly encourage users to try it out and report any issues or feedback. Please refer to the [release schedule](https://opensearch.org/releases.html) for GA release information.
+
+### DEPRECATION NOTICES
+
+**Deprecating support for Ubuntu Linux 20.04**
+Please note that OpenSearch and OpenSearch Dashboards will deprecate support for Ubuntu Linux 20.04 as a continuous integration build image and supported operating system in an upcoming version, as Ubuntu Linux 20.04 will reach end-of-life with standard support as of April 2025 (refer to [this notice](https://ubuntu.com/blog/ubuntu-20-04-lts-end-of-life-standard-support-is-coming-to-an-end-heres-how-to-prepare) from Canonical Ubuntu). For a list of the compatible operating systems, [visit here](https://opensearch.org/docs/latest/install-and-configure/os-comp/).
+
+**Deprecating support for Amazon Linux 2 on OpenSearch Dashboards**
+Please note that OpenSearch Dashboards will deprecate support for Amazon Linux 2 as a continuous integration build image and supported operating system in an upcoming version, as Node.js 18 will reach end-of-life with support as of April 2025 (refer to [this notice](https://nodejs.org/en/blog/announcements/v18-release-announce) from nodejs.org) and newer version of Node.js LTS version (20+) will not support runtime on Amazon Linux 2. For a list of the compatible operating systems, [visit here](https://opensearch.org/docs/latest/install-and-configure/os-comp/).
+
+## Breaking Changes
+* For a full list of breaking changes and deprecated/removed features in version 3.0.0, please see details in the [meta issues](https://github.com/opensearch-project/opensearch-build/issues/5243).
```

Comment:
I think we should list out all breaking changes here than pointing to an issue which then points to sub-issues.